### PR TITLE
Add escalateDisputeToMediator test

### PR DESCRIPTION
**Pull request by ilake**

### CODE REVIEW

File: test/escalateDisputeToMediator.js [hunk 1]
```diff
@@ -3,18 +3,95 @@ const InkProtocol = artifacts.require("./mocks/InkProtocolMock.sol")
 
 contract("InkProtocol", (accounts) => {
   let protocol
+  let buyer = accounts[1]
+  let seller = accounts[2]
+  let unknown = accounts[accounts.length - 1]
 
   beforeEach(async () => {
     protocol = await InkProtocol.new()
```
Comment (ngan) [inline test/escalateDisputeToMediator.js]: Is this needed?

Comment (ngan) [reply test/escalateDisputeToMediator.js]: If not, let’s remove this and the let statement above as well.

File: test/escalateDisputeToMediator.js [hunk 2]
```diff
@@ -3,18 +3,95 @@ const InkProtocol = artifacts.require("./mocks/InkProtocolMock.sol")
...
+        protocol,
+        transaction
+      } = await $util.buildTransaction(buyer, seller, {
+        finalState: $util.states.Disputed
+      })
+
+      let tx = await protocol.escalateDisputeToMediator(transaction.id, { from: seller })
+      transaction = await $util.getTransaction(transaction.id, protocol)
+
+      assert.equal(transaction.state, $util.states.Escalated)
```
Comment (ngan) [inline test/escalateDisputeToMediator.js]: Let’s remove line 89 and 91. The state is actually internal and the getTransaction method just assumes it’s in that state.

**Review by ngan — CHANGES_REQUESTED** [review]

**Comment by ilake** [comment]
@ngan please review this, thanks

**Pull request merged**